docs/_static/eol_banner.js

```diff
@@ -0,0 +1,10 @@
+window.onload = function() {
+    var elem = document.createElement('div');
+    var body = document.getElementsByClassName("bodywrapper")[0]
+    var doc = document.getElementsByClassName("body")[0]
+    elem.className = "admonition important"
+    elem.id = "pulp-2to3-migration-eol-banner"
+    elem.innerHTML = "<p>This plugin reaches its EOL on XXX, the last supported pulpcore" +
```

Comment:
Need to pick a date https://discourse.pulpproject.org/t/pulp-2to3-migration-plugin-end-of-life-eol-date/457